go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/envoyproxy/go-control-plane/xdsmatcher v0.0.0-20230524161521-aaaacbfbe53e
 	github.com/fatih/color v1.14.1
 	github.com/fsnotify/fsnotify v1.6.0
+	github.com/go-mmap/mmap v0.7.0
```

Comment:
If we decide to continue with mmap, I think we should use a better maintained library. https://github.com/edsrzf/mmap-go is an option.

Reply:
This library doesn't have methods like Open, Write, Close and Flush. I am checking how can we use this.

Reply:
@dhiaayachi comments resolved.